.github/actions/flutter-test/action.yml

```diff
@@ -51,6 +51,7 @@ runs:
             # Remove native binding/bridge files from coverage.
             # These FFI/JNI bindings are currently not unit tested due to limitations of FFI/JNI mocking.
             dart run remove_from_coverage -f coverage/lcov.info -r 'binding.dart'
+            dart run remove_from_coverage -f coverage/lcov.info -r 'lib/src/native/sentry_native_channel.dart'
```

Comment:
We can remove this from the coverage so CI doesn't complain.

We will eventually remove the file anyway as soon as we have moved away from method channels completely